setup.py

```diff
@@ -44,6 +44,14 @@
 HERE = Path(__file__).resolve().parent
 
 DEBUG_COMPILE = "DD_COMPILE_DEBUG" in os.environ
+FAST_BUILD = os.getenv("DD_FAST_BUILD", "false").lower() in ("1", "yes", "on", "true")
+if FAST_BUILD:
+    print("WARNING: DD_FAST_BUILD is enabled, some optimizations will be disabled")
+else:
+    print("INFO: DD_FAST_BUILD not enabled")
```

Comment:
Also mind adding to the debug output on L450?

Reply:
Done!